internal/registration/internal/domain/user_registration.go

```diff
@@ -0,0 +1,84 @@
+package domain
+
+import (
+	"fmt"
+	"github.com/google/uuid"
```

Comment:
use `goimports` to sort the imports on save. They must be in this order:
- std
- third-party
- local